app/services/decline_offer.rb

```diff
@@ -1,13 +1,18 @@
 class DeclineOffer
   def initialize(application_choice:)
     @application_choice = application_choice
+    @application_choices = application_choice.application_form.application_choices
```

Comment:
Before Tjimen told me about ended_without_success? i was using it to determine if all the candidates course choices were unsuccessful. It can come out now.